samples/src/baremetal-semihosting/README.md

```diff
@@ -1,21 +1,9 @@
+# Bare-metal semihosting sample
+
 This sample shows how to use semihosting with
 [QEMU Arm System emulator](https://www.qemu.org/docs/master/system/target-arm.html)
 targeting the
 [micro:bit board model](https://www.qemu.org/2019/05/22/microbit/).
 
 It uses the startup code and the linker script file from the GNU Arm Embedded
 Toolchain samples.
```

Comment:
Consider adding a note to say that qemu 5.2.0 crashes on this sample.

Reply:
I only tested with 5.2.0. Maybe other versions display the same bug.

Reply:
I tried 4.2.1 (part Ubuntu 20 distribution) and it freezes. Do we know which version works?

Reply:
The original samples worked for me with QEMU 5.0.0 built from source, on Ubuntu 16.04.

Reply:
Yes, it looks like the sample is broken. I checked Simon's hello.hex file attached to https://bugs.launchpad.net/qemu/+bug/1918302, it works with the latest QEMU 6, but the current version of the sample doesn't. I'll raise a separate issue to fix the sample.